src/r2b2/audit.py

```diff
@@ -249,56 +272,52 @@ def run(self):
         """
 
         self.__reset()
-        print('Beginning Audit...')
-        sample_size = 0
+        click.echo('\n==================\nBeginning Audit...\n==================\n')
+        sample_size = self.min_sample_size
         max_sample_size = self.contest.contest_ballots * self.max_fraction_to_draw
         previous_votes_for_winner = 0
+        curr_round = 0
 
         while sample_size < max_sample_size:
+            curr_round += 1
+            print('\n----------\n{:^10}\n----------\n'.format('Round {}'.format(curr_round)))
+
+            if verbose and curr_round > 1:
+                click.echo('\n+--------------------------------------------------+')
+                click.echo('|{:^50}|'.format('Audit Statistics'))
+                click.echo('|{:50}|'.format(' '))
+                click.echo('|{:<50}|'.format('Minimum Sample Size: {}'.format(self.min_sample_size)))
+                click.echo('|{:<50}|'.format('Maximum Sample Size: {}'.format(max_sample_size)))
+                click.echo('|{:50}|'.format(' '))
+                click.echo('|{:^16}|{:^16}|{:^16}|'.format('Round', 'Stopping Prob.', 'Risk'))
+                click.echo('|----------------|----------------|----------------|')
+                for r in range(1, curr_round):
+                    click.echo('|{:^16}|{:^16}|{:^16}|'.format(r, '{:.12f}'.format(self.stopping_prob_schedule[r - 1]),
+                                                               '{:.12f}'.format(self.risk_schedule[r - 1])))
+                click.echo('+--------------------------------------------------+')
+
             self.next_sample_size()
```

Comment:
Not at all urgent, but maybe at some point in the future we change this to allow auditors to input their desired stopping probabilities for the next round size?

Reply:
(I do see that the method itself permits any parameters, just wondering if such functionality should be included in run at some point in the future.)

Reply:
I agree. 

The implementation of `next_sample_size()` and its interaction in `run()` is on the list of things we have yet to come to a decision on. Last time we all discussed it we were deciding if it should be implemented per audit or globally (like the distribution functions).